fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -254,11 +263,9 @@ public void writeFDBLuceneFileReference(@Nonnull String name, @Nonnull FDBLucene
      * @param value the data to be stored
      */
     public void writeData(long id, int block, @Nonnull byte[] value) {
-        FDBStoreTimer timer = context.getTimer();
-        if (timer != null) {
-            //This may not be correct transactionally
-            timer.increment(FDBStoreTimer.Counts.LUCENE_WRITE_SIZE, value.length);
-        }
+        //This may not be correct transactionally
```

Comment:
I am not sure what potential problem this comment is referring to.

Reply:
I am also not really sure. I just keep the existing comment. Probably @scottfines may have better idea?

Reply:
it's not a big deal, I was referring to the possibility that if you rollback after incrementing this timer, the timer will technically be incorrect. This code doesn't do anything that affects the comment one way or the other.